transports/http/domains/headers.go

```diff
@@ -0,0 +1,65 @@
+package domains
```

Comment:
IMHO there is no need to put it in a separate package, it could be part of the http package.

Reply:
Done